### PR TITLE
Couple of fixes to make "valgrind_memalign" work

### PR DESCRIPTION
**Pull request by ldoktor**
This test depends on broken environment, where "vm.process" was not closed on "vm.shutdown". Let's fix that and tweak it to work properly on s390x.

### CODE REVIEW

File: qemu/tests/valgrind_memalign.py [hunk 1]
```diff
@@ -51,5 +51,6 @@ def valgrind_intall():
 
     error.context("Quit guest and check the process quit normally",
                   logging.info)
-    vm.destroy(gracefully=False)
+    vm.monitor.quit()
```
Comment (clebergnu) [inline qemu/tests/valgrind_memalign.py]: Looks good, there's only one typo in the commit message ("makeing").

File: qemu/tests/cfg/valgrind_memalign.cfg [hunk 2]
```diff
@@ -10,6 +10,8 @@
     Ubuntu:
         valgrind_install_cmd = "apt-get install -y valgrind"
     s390x:
+        # -cpu can't be set in TCG
+        cpu_model = ""
```
[the remark this answers is not in the document]
Comment (ldoktor) [reply qemu/tests/cfg/valgrind_memalign.cfg]: Yes, s390 emulation on x86 does not really work and you can do things that are not possible (something like plugging pcie into pci bus in x86 world)

File: qemu/tests/valgrind_memalign.py (hunk 1 above)
Comment (ldoktor) [reply qemu/tests/valgrind_memalign.py]: Updating.

**Comment by ldoktor** [comment]
@clebergnu fixed the commit message typo

**Comment by clebergnu** [comment]
@ldoktor I got carried away, and thought my ACK was the 2nd (and last) one, and merged it.  To @CongLi and @Hao-Liu , if you find issues here, please forgive and let's either fix it or revert it.

Thanks and sorry again!

**Pull request merged**